### PR TITLE
Adds FlameShock guide section.

### PR DESCRIPTION
**Pull request by periodic**
### Description

Adds a section to the Elemental Shaman guide for Flame Shock.  It shows the total uptime, stack history graph and number of Lava Bolts that needed Flame Shock.

The guide is still a prototype.  This is just filling in one section.

### Testing

- Test report URL: `/report/6tVWMyQP4cb1kmBa/10-Heroic+The+Forgotten+Experiments+-+Kill+(3:58)/Ironshock/standard#prototype`
- Screenshot(s):

Before:
image

After:
image

### CODE REVIEW

**Comment by periodic** [comment]
cc: @awildfivreld

**Comment by awildfivreld** [comment]
When you hover over an uptime section, does it show from-to time? I've found that nice to correlate with other things/the timeline.

File: src/parser/shared/modules/Enemies.ts [hunk 1]
```diff
@@ -218,6 +228,74 @@ class Enemies extends Entities<Enemy> {
...
+        if (event.type === 'remove') {
+          active -= 1;
+        }
+        if (prevTimestamp === null) {
+          prevTimestamp = event.timestamp;
+          return;
+        }
+
+        if (prevActive > 0) {
+          console.log('Adding entry', prevTimestamp, event.timestamp, prevActive);
```
Comment (emallson) [inline src/parser/shared/modules/Enemies.ts]: stray `console.log`

**Review by emallson — CHANGES_REQUESTED** [review]
minor change, but this looks reasonable overall

File: src/parser/shared/modules/Enemies.ts (hunk 1 above)
Comment (periodic) [reply src/parser/shared/modules/Enemies.ts]: Ah, thanks.  Missed that one.

**Review by emallson — APPROVED** [review]

**Pull request merged**